Apps/Sandcastle/gallery/Bing Maps Labels Only.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Use the split property to only show layers on one side of a slider."
```

Comment:
Should the description mention labels?

Reply:
Whoops, missed this one. Copy past error.

Reply:
Okay, now it's ready.